modules/HelpDesk/actions/CheckValidateToClose.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Checking Close Validation Action Class.
+ *
+ * @package   Action
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Adrian Kon <a.kon@yetiforce.com>
+ */
+class HelpDesk_CheckValidateToClose_Action extends \App\Controller\Action
+{
+	/**
+	 * {@inheritdoc}
+	 */
+	public function checkPermission(App\Request $request)
+	{
+		if (!\App\Privilege::isPermitted($request->getModule(), 'EditView', $request->getInteger('record'))) {
```

Comment:
$this->recordModel = Vtiger_Record_Model::getInstanceById($request->getInteger('record'), $request->getModule());
$this->recordModel->isEditable